MpsPowershell/docs/Get-PfBuildAlias.md

```diff
@@ -171,6 +172,36 @@ Accept pipeline input: True (ByValue)
 Accept wildcard characters: False
 ```
 
+### -PageSize
```

Comment:
The magic of Autorest--since I generated this from the latest Swagger, it now has Sarah's fix for the paging properties on the ListBuildAliases API.